scripts/package.json

```diff
@@ -148,7 +148,7 @@
     "ts-node": "^10.9.1",
     "tsup": "^6.7.0",
     "type-fest": "^3.4.0",
-    "typescript": "5.0.4",
+    "typescript": "~4.9.3",
```

Comment:
What's up with this version change? Should this be a separate PR?

Reply:
No i did not touch this actually,  i don't know how it got there, actually it does not matter the version of typescript on scripts project i guess,